sklearn/preprocessing/_data.py

```diff
@@ -3318,6 +3318,8 @@ def _more_tags(self):
 
 def power_transform(X, method="yeo-johnson", *, standardize=True, copy=True):
     """
+    Family of parametric, monotonic transformations to make data more Gaussian-like.
```

Comment:
Just a nitpick:

[PEP 257 for multiline docstrings](https://peps.python.org/pep-0257/#multi-line-docstrings) recommends to put the one-liner directly on the first line of the docstring (used by IDEs and API documentation generation tools for a short description of the function).

```suggestion
    """Parametric, monotonic transformation to make data more Gaussian-like.
```